src/utils.ts

```diff
@@ -18,6 +18,21 @@ export async function getVersionsByDirectory(cwd: string) {
   return new Map(packages.map((x) => [x.dir, x.packageJson.version]));
 }
 
+export const getReleaseMessage = (
+  htmlUrls: string[],
+  tagNames: string[],
+  changesetsSummaries: string[]
+) => `###  🦋  This work has been released in release versions: ${tagNames.join(
```

Comment:
just a nit, but perhaps you want to either say `version(s)` and use other such possible pluralization shortcuts here, or you could pluralize based on array length

also, what about making this message the default, and allowing a supplied parameter for this line of the issue/PR close comment?